docs/rules/consistent-test-it.md

```diff
@@ -1,14 +1,26 @@
 # Have control over `test` and `it` usages (`consistent-test-it`)
 
+💼 This rule is enabled in the following
+[configs](https://github.com/jest-community/eslint-plugin-jest#shareable-configurations):
```

Comment:
also, should it say `config` instead of `configs` when only in a single one?

Reply:
I could go either way on this. My first thought was that it was grammatically correct to use `configs` regardless, since this sentence is separated from the serial list with a colon, and it's clear that the list of configs is variable. If I'm wrong or you still prefer the other way, let me know.

Reply:
I'm not a native speaker, so happy to leave it 👍